modules/installation-ibm-power-vs-config-yaml.adoc

```diff
@@ -87,11 +91,17 @@ If you disable simultaneous multithreading, ensure that your capacity planning a
 //====
 //<6> You can optionally provide the `sshKey` value that you use to access the machines in your cluster.
 //endif::openshift-origin[]
-<4> The cluster network plugin to install. The supported values are `OVNKubernetes` and `OpenShiftSDN`. The default value is `OVNKubernetes`.
-<5> The name of an existing resource group.
-<6> Required. The installation program prompts you for this value.
+<4> The smtLevel specifies the level of SMT to set to the control plane and compute machines. The supported values are 1, 2, 4, 8, 'off' and 'on'. The default value is 8. The smtLevel 'off' sets SMT to off and smtlevel 'on' sets SMT to the default value 8 on the cluster nodes.
```

Comment:
I also don't think the `off` and `on` are in backtics here either when they are meant to be.

Reply:
@kelbrown20 the single quotes for 'off' and 'on' are valid values. To include these values with backticks, can I write them as : `'off'` and `'on'` ? single quotes wrapped in backticks? Please confirm. Thanks.

Reply:
Ohh so in the yaml it would be written like 'off' 'on', is that correct?

Reply:
> so in the yaml it would be written like 'off' 'on', is that correct?
Yes, correct. It should be written that way. FYI : @kelbrown20

Reply:
Gotcha, my apologies for the confusion. I think it would still look good in the back tics like `'off'` `'on'` cuz whenever we have word values they're usually in backtics.

Reply:
@kelbrown20 I have updated the PR to have backticks for above values. Request you to check. Thanks.

Reply:
Updates LGTM! sorry for the confusion here, thank you!